pages/common/mysqld.md

```diff
@@ -0,0 +1,19 @@
+# mysqld
+
+> The MySQL Server.
+
+- Display error log messages:
+
+`mysqld --console`
+
+- Log connections and queries to a file:
```

Comment:
@agnivade why is that preferable? I generally think being explicit is nicer, provided it doesn't make the description too verbose or hard to read (which doesn't seem to be the case here).